platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,19 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, partial=True):
```

Comment:
I'd like to mention it somewhere in our docs since PATCH is a newish verb and we used PUTs to for partial updates in Pulp 2. Leaving it out here is fine though.